scripts/install-kusion.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
 
     # base64 encode
-    content=$(base64 -w 0 "$path")
+    content=$(cat "$path" | base64 | tr -d '\n')
```

Comment:
The change from `base64 -w 0` to `cat | base64 | tr -d '
'` is less efficient as it adds an extra process and pipe. The original `base64 -w 0` is the more direct and performant way to achieve the same result. Consider reverting to the original approach.